src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -1278,19 +1315,13 @@ fixCopyCommand(Archive *fout, PQExpBuffer copyBuf, TableInfo *tbinfo, bool isFro
  * Returns true if table in Babelfish Database is to be dumped with INSERT mode.
  * Currently we dump tables with sql_variant columns with INSERT operations to
  * correctly restore the metadata of the base datatype, which is not directly
- * posible with COPY statements. We also dump sys.babelfish_authid_login_ext
- * with INSERT statements so that if target database already has a login with
- * same name as in the source database, only that INSERT query with fail and won't
- * affect the other entries of the catalog table. 
+ * posible with COPY statements.
  */
-bool bbfIsDumpWithInsert(Archive *fout, TableInfo *tbinfo)
+bool
+bbfIsDumpWithInsert(Archive *fout, TableInfo *tbinfo)
 {
 	return (isBabelfishDatabase(fout) &&
-			(hasSqlvariantColumn(tbinfo) ||
-				pg_strcasecmp(fmtQualifiedDumpable(tbinfo),
-					quote_all_identifiers ?
-					"\"sys\".\"babelfish_authid_login_ext\"" :
-					"sys.babelfish_authid_login_ext") == 0));
```

Comment:
Why are we removing this?

Reply:
Actually this change was previously done to forcefully dump babelfish_authid_login_ext data with INSERT statements so that even if one INSERT fails for existing babelfish init user, other INSERTs should still go through. But now that we have decided to not dump babelfish init user, we can safely dump this table with COPY as well. Basically we don't wan't to pose any restriction on the table data will get dumped for this table.